get_info.php

```diff
@@ -1 +1,200 @@
-<?php?>
+<?php
+require 'db/config.php';
+require 'get_query_generators.php';
+
+try {
+    $method = "";
+    if (isset($_SERVER['REQUEST_METHOD'])) {
+        $query = "";
+        $method = $_SERVER['REQUEST_METHOD'];
+        // Request stored as associative array
+        $data = json_decode(file_get_contents("php://input"), true);
+        switch ($method) {
+            case "GET":
+                $query = handle_get($_GET);
+                break;
+
+            default:
+                throw new Exception("Invalid Request");
+                break;
+        }
+    } else {
+        $error = 'No Request Method Found';
+        throw new Exception($error);
+    }
+} catch (Exception $e) {
+    $result = set_http_response(400);
+    date_default_timezone_set('America/Toronto');
+    error_log(
+        date("Y-m-d h:i:sa") . " : " . $e->getMessage() . "\n",
+        3,
+        "errors.log"
+    );
+    print json_encode($result, JSON_PRETTY_PRINT);
+    exit();
+}
+
+// Attempt to execute sql command and print response in json format.
+// If a sql error occurs, JSON error object.
+try {
+    // connect to the mysql database
+    $conn = new PDO(
+        "mysql:host=$servername;dbname=$database",
+        $username,
+        $password
+    );
+    // set the PDO error mode to exception
+    $conn->setAttribute(PDO::ATTR_ERRMODE, PDO::ERRMODE_EXCEPTION);
+    $stmt = $conn->prepare($query);
+
+    bind_variables($stmt, $_GET);
+    $stmt->execute();
+
+    if ($method == "GET") {
+        //fetch all results
+        $fetched = $stmt->fetchAll(PDO::FETCH_ASSOC);
+        $result['TYPE'] = $_GET["what"] . "_package";
```

Comment:
I'm not thrilled with this... `$_GET["what"]` could be anything...no attempt has been made to sanitize the parameters.

Reply:
I was going to implement sanity checks on the next commit. This PR included more than I anticipated such as SQL injection protection and sanity checks.

Reply:
Okay. Fix the other things and you can commit. We'll do the sanitizing in another PR.